pkg/tokenclaims/tokenclaims.go

```diff
@@ -32,11 +45,34 @@ func (c *CustomClaims) Proto() (*structpb.Struct, error) {
 		ap[i] = int64(c.PrivilegeIDs[i])
 	}
 
+	ces := make((map[string]map[string][]any))
+	if c.CloudEvents != nil {
+		for _, evt := range c.CloudEvents.Events {
+			if _, ok := ces[evt.EventType]; !ok {
+				ces[evt.EventType] = map[string][]any{}
+			}
+
+			source := evt.Source
+			if source == nil {
+				source = &GlobalAttestationPermission
+			}
+
+			if _, ok := ces[evt.EventType][*source]; !ok {
+				ces[evt.EventType][*source] = []any{}
+			}
+
+			for _, id := range evt.IDs {
+				ces[evt.EventType][*source] = append(ces[evt.EventType][*source], id)
+			}
```

Comment:
Can we flip this? I don't love having `GlobalAttestationPermission` as a var because that means any package that is imported in token exchange has the ability to change the value. secondarily it's nice to not have to dereference the pointer every time we use it.
```suggestion
            const GlobalAttestationPermission = "GLOBAL_ATTESTATION_PERMISSION"
            // .
            // .
            // .
			source := GlobalAttestationPermission
			if source != nil {
				source = *evt.Source
			}

			if _, ok := ces[evt.EventType][source]; !ok {
				ces[evt.EventType][source] = []any{}
			}

			for _, id := range evt.IDs {
				ces[evt.EventType][source] = append(ces[evt.EventType][source], id)
			}
```